spec/factories/records.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :record do
+    name { Faker::Movies::HitchhikersGuideToTheGalaxy.character }
+    description { Faker::Movies::HitchhikersGuideToTheGalaxy.quote }
+    value { Random.new.rand(0.0..9.9) }
+  end
```

Comment:
```suggestion
 
    user
  end
```

Reply:
? Do you mean to create a user relationship with this?

Reply:
É que tem um `belongs_to` pra `user` dentro de `record` não??

Reply:
Ai quando tu for criar um `record` pela factory não precisa passar um `user` mas se passar ele usa o que tu passo e não cria outro dai.

Reply:
Certo mano.